plan/cbo_test.go

```diff
@@ -34,6 +35,32 @@ var _ = Suite(&testAnalyzeSuite{})
 type testAnalyzeSuite struct {
 }
 
+// this function tests the plan with stats that only have count info
```

Comment:
this -> This, add dot at the end.

Reply:
Could you add some comments to explain why tables does not have stats info?

Reply:
@shenli  I have closed auto analyze